### DISCUSSION TITLE
[FR] Actors import/export

### DISCUSSION THREAD

**Discussion by librafrog** [SubtitleEdit/subtitleedit#7806] [Ideas]
I guess that's only a minor whim, but if is liked, it may be implemented in near future. Just to clarify:

- [ ] R.M 🖱️ » (in the menu Set\Remove Actor) » **Import\Export** mainly in ASSA style

**Comment by niksedk** [comment]
Actors, are only part of actual lines, so it's not possible to "Import actors".
